modules/mono/mono_gc_handle.h

```diff
@@ -49,7 +49,9 @@ struct GCHandleIntPtr {
 	_FORCE_INLINE_ bool operator==(const GCHandleIntPtr &p_other) { return value == p_other.value; }
 	_FORCE_INLINE_ bool operator!=(const GCHandleIntPtr &p_other) { return value != p_other.value; }
 
+#if not defined(__cplusplus) or (__cplusplus < 202002L)
 	GCHandleIntPtr() = delete;
+#endif
```

Comment:
Is this no longer supported in C++ 20?

Reply:
dcl.init.aggr

`no user-declared or ... constructors ` (Until C++11, Since C++20)

Adding a user defined constructor would make this class non-trivial.
Adding an initializer would make this class non-trivial.
This seems like a minimal change.

Reply:
@neikeq I imagine the question is what hard constraints are you looking for on GCHandleIntPtr?
For example "It needs to be trivial/aggregate"